tgui/packages/tgui/interfaces/Reagents.js

```diff
@@ -8,6 +8,8 @@ const bookmarkedReactions = new Set();
 
 const matchBitflag = (a, b) => (a & b) && (a | b) === b;
 
+let page = 1;
```

Comment:
You should be using `useLocalState` for pagination, because assignment to a global variable does not trigger a re-render.

I can exceptionally allow it, but then at least add a comment that this is bad and shouldn't be copied.